src/datastore_mad/remotes/lxd_downloader.sh

```diff
@@ -294,6 +295,22 @@ rm /dev/random /dev/urandom
 EOC
 )
     ;;
+*alt*)
+    terminal="/bin/bash"
+    commands=$(cat <<EOC
+export PATH=/sbin:/usr/sbin:/bin:/usr/bin
+
+rm -f /etc/resolv.conf >> /var/log/chroot.log 2>&1
+echo "nameserver $DNS_SERVER" > /etc/resolv.conf
+
+[ ! -e /dev/random ] && mknod -m 666 /dev/random c 1 8  >> /var/log/chroot.log 2>&1
+[ ! -e /dev/urandom ] && mknod -m 666 /dev/urandom c 1 9  >> /var/log/chroot.log 2>&1
+
+apt-get update >> /var/log/chroot.log 2>&1
+apt-get install $PKG_ALT -y >> /var/log/chroot.log 2>&1
+
+rm /dev/random /dev/urandom
+EOC
 *)
```

Comment:
add ;; after EOC

Reply:
Not only ';;' but closing parenthesis too :(